datalab/mlalpha/_cloud_models.py

```diff
@@ -51,7 +43,9 @@ def _retrieve_models(self, page_token, page_size):
     page_token = list_info.get('nextPageToken', None)
     return models, page_token
 
-  def __iter__(self):
+  def get_iterator(self):
+    """Get iterator of models so it can be used as "for model in Models().get_iterator()".
+    """
     return iter(datalab.utils.Iterator(self._retrieve_models))
 
   def get_model_details(self, model_name):
```

Comment:
Now I think it is better to return whatever service returns, so it is consistent across the methods.